core/constants.ts

```diff
@@ -0,0 +1,56 @@
+// Constants that will be shared by the wasm core of the emulator
+// And libraries built around the wasm (such as the official JS), or @CryZe wasmboy-rs
+
+// ----------------------------------
+// Wasmboy Memory Map
+// https://docs.google.com/spreadsheets/d/17xrEzJk5-sCB9J2mMJcVnzhbE-XH_NvczVSQH9OHvRk/edit?usp=sharing
+// ----------------------------------
+
+// WasmBoy
+export const WASMBOY_MEMORY_LOCATION: i32 = 0x000000;
+export const WASMBOY_MEMORY_SIZE: i32 = 0x8b0000;
+export const WASMBOY_WASM_PAGES: i32 = WASMBOY_MEMORY_SIZE / 1024 / 64;
+
+// AssemblyScript
+export const ASSEMBLYSCRIPT_MEMORY_LOCATION: i32 = 0x000000;
+export const ASSEMBLYSCRIPT_MEMORY_SIZE: i32 = 0x000400;
+
+// WasmBoy States
+export const WASMBOY_STATE_LOCATION: i32 = 0x000400;
+export const WASMBOY_STATE_SIZE: i32 = 0x000400;
+
+// Gameboy Internal Memory
+export const GAMEBOY_INTERNAL_MEMORY_LOCATION: i32 = 0x000800;
+export const GAMEBOY_INTERNAL_MEMORY_SIZE: i32 = 0x00ffff;
```

Comment:
Hi, I spotted an odd size :)

Reply:
Hello! Dude thanks for looking through the changes :)

Yeah I meant to capitalize those f's, but the size is indeed 64KB 😄 since the gameboy had a bunch of switchable banks and things

Reply:
Just wondered because 64KB is 0x010000 :)

Reply:
Ahhh, yeah I always just subtract 1. 😅  Well then the GB memory is 0xFFFF, and I mean, it works 🤷‍♂️